ZeeHomeScreen/Classes/Components/SectionComposite/SectionCompositeViewController.swift

```diff
@@ -36,6 +37,7 @@ import Zee5CoreSDK
     var displayLanguage: String?
     var contentLanguages: String?
     var liveComponents = [ComponentModelProtocol]()
+    private var adsViewControllers: [BannerCellViewController] = []
```

Comment:
Please remove this, we don't want to keep a cache of view controllers. At least, not in SectionCompositeViewController.